apps/ideas/src/components/Card/IdeaCardList.js

```diff
@@ -1,71 +1,102 @@
-import React from 'react'
-import CircularIndeterminateLoader from '../Loader/CircularIndeterminateLoader'
-import IdeaCard from './IdeaCard'
-import StyledIdeaCards from './StyledIdeaCards/StyledIdeaCards'
+import React from 'react';
+import CircularIndeterminateLoader from '../Loader/CircularIndeterminateLoader';
+import IdeaCard from './IdeaCard';
+import StyledIdeaCards from './StyledIdeaCards/StyledIdeaCards';
+import { TextCardTitleBox } from './StyledIdeaCard';
 import StyledCardsWrapper from '../CardsWrapper/StyledCardsWrapper';
-import axios from "axios";
-import { env } from "../../utils/EnvironmentVariables";
+import axios from 'axios';
+import { env } from '../../utils/EnvironmentVariables';
+import ProjectFilter from './ProjectFilter/ProjectFilter';
 import Grid from '@mui/material/Grid';
 
-
-
 function IdeaCardList() {
   const [cards, setCards] = React.useState([]);
   const [selectedCard, setSelectedCard] = React.useState({});
   const [loading, setLoading] = React.useState(true);
-
+  const [sortedCards, setSortedCards] = React.useState([]);
+
+  const sortingConfigs = [
+    { value: 'default', label: '' },
+    {
+      value: 'descending:mostRecentCommentTime',
+      label: 'Recent Activity',
+    },
+    {
+      value: 'descending:updated_at',
+      label: 'Recent Ideas',
+    },
+    {
+      value: 'ascending:hourCommitmentMin',
+      label: 'Time Commitment',
+    },
+  ];
+
+  const handleSetSortedCards = (sortedCardList) => {
```

Comment:
I think this function could be removed, and instead `setSortedCards` could be passed directly into the `handleSetSortedCards` prop of the `ProjectFilter` component

Reply:
I removed this function and it seems to work fine. I also renamed handleSetSortCards to handleSetSortedElements on the SortableDropdown component to make it more general.